tests/integration/src/test/resources/bug-login-breaks-stream.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
```

Comment:
don't create an empty config just because. Use one of the existing configs.... On a second thought, you shouldn't need a config at all to reproduce this issue